cmds/record.c

```diff
@@ -1474,8 +1474,10 @@ static void print_child_usage(struct rusage *ru)
 
 #define UFTRACE_MSG  "Cannot trace '%s': No such executable file.\n"
 
-#define MCOUNT_MSG  "Can't find '%s' symbol in the '%s'.\n"		\
-"\tIt seems not to be compiled with -pg or -finstrument-functions flag\n" 	\
+#define MCOUNT_MSG  "Cannot trace '%s' symbol in the '%s'.\n" \
+"\tusing full dynamic tracing which is -P option works without" \
+ " -pg or -finstrument-functions.\n" \
+"\totherwise, It seems need to be compiled with -pg or -finstrument-functions flag\n" 	\
```

Comment:
I still see that the first line should use 'find' instead of 'trace' since we don't trace "mcount" itself.  How about this?

Cannot find '%s' symbol in the '%s'.\n
It seems not to be compiled with -pg or -finstrument-functions flag.\n
You can rebuild your program with it or use -P option for dynamic tracing.

Reply:
sorry, there was something wrong.
after fixing I didn't commit again.
I fixed it.
hoping all of proccesses has done correctly in this time. :)
Thank you.